A32NX/html_ui/Pages/A32NX_Core/A32NX_Core.js

```diff
@@ -34,15 +102,32 @@ class A32NX_Core {
             return;
         }
 
+        const startTime = ENABLE_TOTAL_UPDATE_TIME_TRACING ? Date.now() : 0;
         this.updateACPowerStateChange();
 
         const deltaTime = this.getDeltaTime();
 
         this.soundManager.update(deltaTime);
-        this.modules.forEach(module => {
-            module.update(deltaTime, this);
+
+        let updatedModules = 0;
+        this.modules.forEach(moduleDefinition => {
+            const moduleDeltaTime = this.moduleThrottlers[moduleDefinition.name].canUpdate(deltaTime);
+
+            if (moduleDeltaTime !== -1) {
+                moduleDefinition.module.update(moduleDeltaTime, this);
```

Comment:
Do you think it would be a good idea to limit it to one module running per frame? Might help remove occasional worst case stutters.

Reply:
We could look into that, right now from my very limited testing this improves stuttering a bit.